config/routes.rb

```diff
@@ -0,0 +1,26 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  scope :api, defaults: { format: :json } do
+    scope :v1 do
+      devise_for :users, path: '', path_names: {
+        sign_in: 'login',
+        sign_out: 'logout',
+        registration: 'signup'
+      },
+      controllers: {
+        sessions: 'api/v1/users/sessions',
+        registrations: 'api/v1/users/registrations'
+      }
+    end
+  end
+  
+  namespace :api do
+    namespace :v1 do
+      resources :cars
+      resources :users do
+        resources :reservations
```

Comment:
- Why don't use the `only:` statement here? this could help to avoid leaving an open endpoint without using it. I kindly recommend you add this to the endpoints. 🐱